deploy/cluster-monitoring-config/sss-config.yaml

```diff
@@ -0,0 +1,2 @@
+matchLabels:
+  api.openshift.com/techpreview/uwm: "false"
```

Comment:
Will this be false on all ClusterDeployments or missing?

Reply:
i'll get a change ready make it false on all clusters by default.
i think it wont work otherwise